docs/changelog.rst

```diff
@@ -7,6 +7,7 @@ CHANGELOG
 
 **Minor Changes**
 
+- Add settings allowing to remove menu on the top left
```

Comment:
Not the entire menu -> "Add a settings allowing to remove certain items from the left menu"